### PR TITLE
ci: setup linting tooling

### PR DESCRIPTION
**Pull request by Jefiozie**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Comment by SanderElias** [comment]
Is this coming along?

**Comment by Jefiozie** [comment]
> Is this coming along?

Yes, I hope to finish this soon, removed somethings that shouldn't be removed.

File: core/cli/src/lib/utils/procesmanager/TaskWorker.ts [hunk 1]
```diff
@@ -88,7 +88,7 @@ export class TaskWorker {
 
   #debounceClean: NodeJS.Timeout | undefined;
   #clean = ({ source, msg }: { source: string; msg: any }) => {
-    if (!!!msg) {
+    if (!msg) {
       /** no message means a clean exit from the worker, don't restart */
```
Comment (SanderElias) [inline core/cli/src/lib/utils/procesmanager/TaskWorker.ts]: Could you please tell me why you removed the explicit cast to boolean? I recall I put it there for an edge case, but I can't recall the case tho.

**Review by SanderElias — APPROVED** [review]
LGTM

File: core/cli/src/lib/utils/procesmanager/TaskWorker.ts (hunk 1 above)
Comment (Jefiozie) [reply core/cli/src/lib/utils/procesmanager/TaskWorker.ts]: In the first lint run I think it was removed, there everything was realllly strict / explicit. But I added it back

**Pull request merged**